app/services/users/manage_credits.rb

```diff
@@ -0,0 +1,50 @@
+module Users
+  class ManageCredits
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def initialize(user, user_params)
+      @user = user
+      @organization_id = user_params[:organization_id]
```

Comment:
Instead of using all this variables, what do you think of use just `@user_params = user_params`

And in each place you access what you need.